### DISCUSSION TITLE
issue: [!] undefined method `get_profiles' for nil:NilClass

### DISCUSSION THREAD

**Discussion by MoustafaHegazy1** [fastlane/fastlane#21769] [General] [answered]
we are facing this issue while installing last version 2.218.0 

<img width="1430" alt="Screenshot 2024-01-03 at 7 03 16 PM" src="https://github.com/fastlane/fastlane/assets/150604550/649de7fb-a525-4f8f-acea-ae9a726ba44c">

which leads to our pipeline to fail so 
for now i'm not using latest i'm using 2.217.0

**Comment by dbeltram** [comment, marked as answer]
https://github.com/fastlane/fastlane/issues/21762

**Reply by MoustafaHegazy1** [reply]
Many Thanks 🙏
